src/itest/java/org/kiwiproject/consul/AgentClientITest.java

```diff
@@ -26,6 +26,7 @@
 import org.kiwiproject.consul.model.health.ServiceHealth;
 import org.kiwiproject.consul.option.ImmutableQueryOptions;
 import org.kiwiproject.consul.option.ImmutableQueryParameterOptions;
+import org.kiwiproject.consul.option.Options;
 import org.kiwiproject.consul.option.QueryOptions;
```

Comment:
do you still need to import `QueryOptions`?

Reply:
yes, b/c it's used on line 412:

```java
var blockingQueryOptions = QueryOptions.blockSeconds(ttl, service.getResponse().getContentHash()).build();
```